_posts/2023-06-29-run-and-debug-rust-verification-harnesses-natively-on-vscode.md

```diff
@@ -0,0 +1,170 @@
+---
+layout: post
+title:  "Run and debug Rust verification harnesses from VS Code"
+---
+
+Kani is a verification tool that can help you systematically test properties about your Rust code.
+To learn more about Kani, check out the [Kani tutorial](https://model-checking.github.io/kani/kani-tutorial.html) and our [previous blog posts](https://model-checking.github.io/kani-verifier-blog/).
+
+## Introducing the Kani VS Code Extension
+
+We are delighted to introduce the [Kani VS Code Extension](https://marketplace.visualstudio.com/items?itemName=model-checking.kani-vscode-extension), which is now available on the VS Code marketplace.
+As you write Kani harnesses, the extension detects them and conveniently displays them within your testing panel.
+The extension offers detailed diagnostics, feedback about verification failures, error messages, and stack traces.
```

Comment:
maybe "There you will find" instead of "The extension offers", since we're talking about the display in the panel, no?

Reply:
So, these features are not in the testing panel, so "there you will find" would not be accurate